### PR TITLE
320 test read big integer

### PR DESCRIPTION
**Pull request by RostislavKrotenko**

### CODE REVIEW

**Review by DappY-127 — DISMISSED** [review]

**Review by kovalchuk-vita — APPROVED** [review]

File: src/test/java/org/codewars/utils/ReaderTest.java [hunk 1]
```diff
@@ -1,14 +1,23 @@
 package org.codewars.utils;
 
 import org.testng.Assert;
+
+import org.testng.Reporter;
```
Comment (DappY-127) [inline src/test/java/org/codewars/utils/ReaderTest.java:5]: I didn't see where it is used and why

Comment (RostislavKrotenko) [reply src/test/java/org/codewars/utils/ReaderTest.java]: Thanks, I have forgotten to remove it

File: src/test/java/org/codewars/utils/ReaderTest.java [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.*;
+import java.math.BigInteger;
+import java.util.NoSuchElementException;
+
+import static org.testng.Assert.*;
```
Comment (DappY-127) [inline src/test/java/org/codewars/utils/ReaderTest.java:14]: re-import
import static org.testng.Assert; imports the Assert class from the org.testng package. And it already exists above.

Comment (RostislavKrotenko) [reply src/test/java/org/codewars/utils/ReaderTest.java]: already removed it

**Review by DappY-127 — APPROVED** [review]

**Review by kovalchuk-vita — APPROVED** [review]

**Pull request merged**